### PR TITLE
adding a reflex logo component

### PR DESCRIPTION
**Pull request by tgberkeley**
### All Submissions:

- [ ] Have you followed the guidelines stated in [CONTRIBUTING.md](https://github.com/reflex-dev/reflex/blob/main/CONTRIBUTING.md) file?
- [ ] Have you checked to ensure there aren't any other open [Pull Requests](https://github.com/reflex-dev/reflex/pulls ) for the desired changed?

### Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### New Feature Submission:

- [ ] Does your submission pass the tests? 
- [ ] Have you linted your code locally prior to submission?

### Changes To Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### **After** these steps, you're ready to open a pull request.

    a. Give a descriptive title to your PR.

    b. Describe your changes.

    c. Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).

### CODE REVIEW

File: reflex/components/datadisplay/logo.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+class Logo(Component):
+
+    @classmethod
+    def create(cls, **props,):
+        return rx.center(
+        rx.link(
+            rx.hstack(
+                "Built with ",
+                rx.image(src="https://raw.githubusercontent.com/reflex-dev/reflex-web/main/assets/Reflex.svg"),
```
Comment (Lendemor) [inline reflex/components/datadisplay/logo.py:12]: I'd rather we add the logo in the source than using a link.

Comment (Lendemor) [inline reflex/components/datadisplay/logo.py:12]: I'd rather we add the logo in the source than using a link.

**Review by Lendemor — CHANGES_REQUESTED** [review]

File: reflex/components/datadisplay/logo.py (hunk 1 above)
Comment (picklelo) [inline reflex/components/datadisplay/logo.py:12]: We can do `rx.color_mode_cond` and show the white logo in dark mode: https://github.com/reflex-dev/reflex-web/blob/main/assets/Reflex_white.svg

Comment (picklelo) [reply reflex/components/datadisplay/logo.py:12]: We currently don't have a way to bundle up assets with a component (unless we include the logos in every app by default, which we probably don't want)

Comment (Lendemor) [reply reflex/components/datadisplay/logo.py:12]: Yeah but using a link means we'll break some version of Reflex in the future if we ever move the image somewhere.
Could we make a NPM package that is a dependency of Logo ? 
This way the image would be only installed when the component is used?

Comment (tgberkeley) [reply reflex/components/datadisplay/logo.py:12]: Put it into html and then inside a color mode cond

Comment (tgberkeley) [reply reflex/components/datadisplay/logo.py]: done

Comment (tgberkeley) [reply reflex/components/datadisplay/logo.py]: done

File: reflex/components/datadisplay/logo.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+                ),
+                text_align="center",
+                align="center",
+                padding="1em",
+            ),
+            href="https://reflex.dev",
+            size="3",
+        ),
+        **props,
+        width="100%",
```
Comment (masenf) [inline reflex/components/datadisplay/logo.py:48]:
```suggestion
        width=props.pop("width", "100%"),
        **props,
```

Avoid an issue if the user tries to pass a custom width in the props

**Review by masenf — APPROVED** [review]

**Review by Lendemor — APPROVED** [review]

**Pull request merged**